src/core/operations/Sort.mjs

```diff
@@ -131,6 +133,48 @@ class Sort extends Operation {
         return a.localeCompare(b);
     }
 
+    /**
+     * Comparison operation for sorting of hexadecimal values.
+     *
+     * @author Chris van Marle
+     * @private
+     * @param {string} a
+     * @param {string} b
+     * @returns {number}
+     */
+    static _hexadecimalSort(a, b) {
+        const a_ = a.split(/([^\da-f]+)/i),
+            b_ = b.split(/([^\da-f]+)/i);
+
+        for (let i = 0; i < a_.length; ++i) {
```

Comment:
Minor: consider using `Array.map` instead for `for` loops.

Reply:
Sure, updated in upcoming PR.